Source/Widgets/BaseLayerPicker/BaseLayerPicker.css

```diff
@@ -22,25 +25,32 @@
     border-radius: 10px;
     -moz-user-select: none;
     -webkit-user-select: none;
+    -ms-user-select: none;
     user-select: none;
     -webkit-transform: translate(0, -20%);
     -moz-transform: translate(0, -20%);
+    -ms-transform: translate(0, -20%);
+    -o-transform: translate(0, -20%);
     transform: translate(0, -20%);
     visibility: hidden;
     opacity: 0;
     -webkit-transition: visibility 0s 0.2s, opacity 0.2s ease-in, -webkit-transform 0.2s ease-in;
     -moz-transition: visibility 0s 0.2s, opacity 0.2s ease-in, -moz-transform 0.2s ease-in;
+    -o-transition: visibility 0s 0.2s, opacity 0.2s ease-in, transform 0.2s ease-in;
```

Comment:
I'm not sure and a quick Google didn't give an answer.  We already had the `-o-` prefix in a bunch of places, and since by default the tool adds them I figured it was better to be consistent.  If you are convinced we don't need them, we should remove them all, including the ones that were already there (and next time I use Web Essentials I'll just tell it we don't care about Opera).